controller/users.go

```diff
@@ -278,6 +287,22 @@ func (c *UsersController) createUserInDB(ctx *app.CreateUsersContext, identityID
 	return identity, user, nil
 }
 
+func (c UsersController) deleteUser(ctx context.Context, userID uuid.UUID, identityID uuid.UUID) error {
```

Comment:
Worth moving it to authentication/account/service/user.go
Something like `HardDeleteUser(ctx context.Context, identity Identity) error`. We will need to start moving the existing business logic from user(s) controllers to services later but for now let's add new stuff to services.

Reply:
Addressed in 72b25766ca1fbc16a01c54b7e801702037a4b174